docs/products/kafka/karapace/concepts/kafka-rest-proxy-authorization.rst

```diff
@@ -0,0 +1,11 @@
+Apache Kafka® REST proxy authorization
+======================================
+The Apache Kafka® REST proxy authorization, when :doc:`enabled <../howto/enable-karapace>` on your Aiven for Apache Kafka® service, allows the delegation of user authentication and authorization to Kafka. Apache Kafka® REST proxy authorization uses the authorization rules defined in :doc:`Apache Kafka® ACLs </docs/products/kafka/concepts/acl>`. 
```

Comment:
This exact phrase is present in the Kafka concept, should we try to rephrase to avoid page SEO cannibalism

Reply:
reworded this a bit.